### PR TITLE
added contributors in acknowledgements

### PR DESCRIPTION
**Pull request by 0xMuluh**

### CODE REVIEW

File: inst/pages/acknowledgments.qmd [hunk 1]
```diff
@@ -151,6 +151,26 @@ set enrichment analysis (MSEA) and multi-omics prediction and classification.
 - *Yihan Liu*, assisted Dr. Mallick in writing the chapters on meta-anlayses,
 MSEA and multi-omics prediction and classification.
 
+- *Jennifer Wokaty*
+
+- *Hervé Pagès*
```
Comment (antagomir) [inline inst/pages/acknowledgments.qmd]: remove (just technical submission issue)

File: inst/pages/acknowledgments.qmd [hunk 2]
```diff
@@ -151,6 +151,26 @@ set enrichment analysis (MSEA) and multi-omics prediction and classification.
...
+
+- *Danielle Callan*
+
+- *Katariina Pärnänen*
+
+- *Yu Gao 高煜* 
+
+- *Karoline*
```
Comment (antagomir) [inline inst/pages/acknowledgments.qmd]: -> Karoline Faust

**Review by antagomir — COMMENTED** [review]
Also add: Jesse Pasanen

File: inst/pages/acknowledgments.qmd (hunk 1 above)
Comment (0xMuluh) [reply inst/pages/acknowledgments.qmd]: ACK.

**Review by antagomir — APPROVED** [review]

**Pull request merged**